runtime/src/accounts.rs

```diff
@@ -547,14 +546,21 @@ impl Accounts {
                     })
                     .map(|(pubkey, account, _slot)| (*pubkey, account.lamports))
                 {
-                    collector.push(data)
+                    let index_of_first_smaller_account =
+                        collector.iter().position(|&(_, balance)| balance <= data.1);
```

Comment:
Sure, that works. I deliberately ignored, since scan_accounts methods typically don't guarantee order, but I can see the value for the `num` boundary.

Reply:
Especially since this one is for whale watching.  The people who pay any attention to it are the same ones that will spin a yarn over any slight discrepancy :laughing: